gulpfile.js

```diff
@@ -264,9 +275,13 @@ function listPostBuildTasks(folder, done) {
 
 function post_build(arch, folder, done) {
 
-    if ((arch =='linux32') || (arch == 'linux64')) {
+    if ((arch === 'linux32') || (arch === 'linux64') || (arch === 'armv7')) {
         // Copy Ubuntu launcher scripts to destination dir
         var launcherDir = path.join(folder, pkg.name, arch);
+        if (arch === 'armv7') {
```

Comment:
I don't know too much about `arm`, does it need the ubuntu launcher scripts? If I'm not wrong you are generating a zip file and not an installer. Does these scripts work under arm environment?

Reply:
I've included them as a template for users to create their own desktop files to launch the Configurator, but upon second thought they should be removed - users can still create their own launchers and/or simply check the repo for a base to start with.